### ISSUE TITLE
feature: values of get_id (Meter identification, reported protocol, baud) as diagnostic sensors

### ISSUE THREAD

**Issue by webmasterkai** [aquaticus/esphome-iec62056#3]
In the log output I noticed some info that look to be the result of `ESP_LOGD(TAG, "Meter identification: '%s'", p);`, `ESP_LOGD(TAG, "Meter reported protocol: %c", (char) mode_);`, and `ESP_LOGD(TAG, "Switching to new baud rate %u bps ('%c')", new_baudrate, baud_rate_char);`. It might be nice to offer these as diagnostic sensors that can be displayed and updated in the home assistant interface. Is this possible?

**Comment by aquaticus** [comment]
It's possible. I remember that at one point during development, there was a diagnostic sensor. I removed it because its usability seemed limited to the initial setup, when you're most likely to monitor the debug log. Later on, you're more concerned with whether you're getting readings or not. In practice, the meter identification and protocol will remain the same at all times.

**Comment by webmasterkai** [comment]
I agree, the utility is limited. Still, it might be nice to have the option for it to be a sensor.

**Comment by aquaticus** [comment]
It is worth considering for the next software release.
